Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM ubuntu:16.04 
-ENV pythonVersions='python2.7 python3.5'
+FROM ubuntu:16.10
+ENV python2Versions='python2.7'
+ENV python3Versions='python3.5 python3.6'
+ENV pythonVersions="$python2Versions $python3Versions"
 
 RUN set -x \
     && apt-get update \
     && apt-get install -y --no-install-recommends $pythonVersions \
+    && apt-get install -y python3-venv $(for py3ver in $python3Versions; do echo $py3ver-venv; done) \
```

Comment:
+1. I would think `python3-venv` results in the current default python3 version of the Linux distribution (probably 3.5 now and 3.6 in a next release)